templates/ib/scenes-swift4.stencil

```diff
@@ -81,6 +81,21 @@ import {{module}}
     }
     return controller
   }
+
+  @available({% if isAppKit %}OSX 10.15{% else %}iOS 13.0, tvOS 13.0{% endif %}, *)
+  {{accessModifier}} func instantiate(creator block: @escaping (NSCoder) -> T?) -> T {
+    let identifier = {% if isAppKit %}NSStoryboard.SceneIdentifier({% endif %}self.identifier{% if isAppKit %}){% endif %}
+    {% if isAppKit %}
+    guard let controller = storyboard.storyboard
+      .__instantiate{{controller}}(withIdentifier: identifier, creator: block) as? T else {
```

Comment:
I don't think a `guard` with `as` cast is needed? The new `instantiate` methods are generic, and Swift should be able to infer the type to be `T`. So I think the code should just be what you have for non-appkit platforms, namely:

```swift
return storyboard.storyboard.instantiate{{controller}}(identifier: identifier, creator: block)
```